File: CB.EN.U4CYS22057/30_03_2023/python/number_swap.py
# This program swpas the values of two variables without a third variable
# Declaring variables
# Getting the value of and variables
print("Enter the value of first variable")
a = int(input())
print("Enter the value of second variable")
b = int(input())
print("The numbers before swapping are " + str(a) + " and " + str(b))

# Swapping the variables
a = a + b
b = a - b
a = a - b

# Final output
print("The numbers after swapping are " + str(a) + " and " + str(b))

File: CB.EN.U4CYS22057/30_03_2023/c/number_swap.c
#include <stdio.h>

int main() {
    int a, b;

    // Getting the value of and variables
    printf("Enter the value of first variable: ");
    scanf("%d", &a);
    printf("Enter the value of second variable: ");
    scanf("%d", &b);
    printf("The numbers before swapping are %d and %d\n", a, b);

    // Swapping the variables
    a = a + b;
    b = a - b;
    a = a - b;

    // Final output
    printf("The numbers after swapping are %d and %d\n", a, b);
    return 0;
}
